scripts/train.py

```diff
@@ -187,8 +207,15 @@ def main(cfg):
     # Train the model.
     ######################################################################
 
-    trainer.fit(model, train_loader, [train_loader, val_loader, unseen_loader])
+    trainer.fit(model, train_loader, [train_loader_eval, val_loader, unseen_loader])
 
 
 if __name__ == "__main__":
-    main()
+    parser = argparse.ArgumentParser()
```

Comment:
is this necessary? can't it come from the dataset/model combo?

Reply:
you can modify the inputs directly in the command line with configs.

https://hydra.cc/docs/advanced/override_grammar/basic/